src/Microsoft.ML.LightGBM/LightGbmStatic.cs

```diff
@@ -114,11 +109,100 @@ public static partial class ClassificationTrainers {
 
             return rec.Output;
         }
-    }
 
-    internal static class LightGbmStaticsUtils {
+        /// <summary>
+        /// LightGbm <see cref="RankingContext"/> extension method.
```

Comment:
>LightGbm <see cref="RankingContext"/> extension method. [](start = 12, length = 55)

Change the summary to look like the below summary

Reply:
still here

---
In reply to: [227866657](https://github.com/dotnet/machinelearning/pull/1352#discussion_r227866657) [](ancestors = 227866657)